src/coreclr/jit/emit.cpp

```diff
@@ -3864,6 +3869,21 @@ void emitter::emitDispGCinfo()
     printf("\n\n");
 }
 
+void emitter::emitDispJumpList()
+{
+    printf("Emitter Jump List:\n");
+    unsigned int  jmpCount    = 0;
+    instrDescJmp* jmp         = emitJumpList;
+    instrDescJmp* previousJmp = nullptr;
```

Comment:
unused?

Reply:
Yup, good catch. Also converted to for loop as suggested.